apps/klaviyo/src/pages/api/manifest.ts

```diff
@@ -12,27 +12,29 @@ const handler = createManifestHandler({
     const { appBaseUrl } = context;
 
     return {
+      about:
+        "Klaviyo integration allows to send Saleor events to Klaviyo, where notifications can be sent to customers.",
```

Comment:
```suggestion
        "Klaviyo integration allows sending Klaviyo notifications on Saleor events.",
```

because we have "send" twice